cypress/integration/pages/articles/tests.js

```diff
@@ -146,15 +146,19 @@ export const testsThatFollowSmokeTestConfig = ({
               cy.get('figure')
                 .eq(0)
                 .then($fig => {
-                  if ($fig.find('p').length > 0) {
-                    cy.get('figure p')
-                      .eq(0)
-                      .should('contain', copyrightHolder);
+                  if (copyrightHolder === 'BBC') {
+                    cy.get($fig).within(() => {
+                      cy.get('p[class^="Copyright"]').should('not.be.visible');
+                    });
```

Comment:
I'm not sure if the additional call to `cy.get` and use of `$fig` is needed here. Can it be written more concisely something like this?

```javascript
cy.get('figure')
  .eq(0)
  .within(() => {
    if (copyrightHolder === 'BBC') {
      cy.get('p...
    ...
  });
```

ref. https://docs.cypress.io/api/commands/get.html#Get-in-within

Reply:
Also I don't think relying on the `class` is a good idea as these are automatically generated and could change (see https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements)